src/components/TokenLocking/MilesReceipt.tsx

```diff
@@ -25,68 +25,75 @@ const MilesReceipt = ({
 
   return (
     <Modal open={open} onClose={onClose} slotProps={{ backdrop: { sx: { backgroundColor: '#636669BF' } } }}>
-      <div className={css.milesReceipt}>
+      <>
         <div className={clsx(css.gradientBackground, { [css.unlockGradient]: isUnlock })} />
-        <Stack className={css.leftReceipt}>
+        <Stack className={css.milesReceipt}>
           <SvgIcon component={SafeExplorers} inheritViewBox sx={{ width: '282px', height: 'auto' }} />
-          <Stack mt="auto">
-            <Typography variant="h1" component="div" fontWeight="bold" mb={1}>
-              {isUnlock ? 'Your tokens are unlocked' : 'Welcome on board'}
+          <Stack mt="auto" className={css.topReceipt}>
+            <Typography variant="h1" component="div" fontWeight="bold" mb={1} mt={7}>
+              {isUnlock ? 'Your tokens are unlocked' : 'Locking started...'}
```

Comment:
Should we also rephrase the unlocking to "Unlocking started..." or something similar?